File: hackerrank/practice/love-letter-mystery.cpp
#include <iostream>
#include <cmath>

using namespace std;

int solve(string a){
	int ans = 0;
	int l = a.length();
	for (int i = 0; i < l/2; i += 1){
		ans += abs(a[i] - a[l-i-1]);
	}
	return ans;
}

int main(void){
	int t; 
	string a;
	cin >> t;
	while (t--){
		cin >> a;
		cout << solve(a) << endl;
	}
}

File: hackerrank/practice/love-letter-mystery.py
class string(str):
	def __sub__(self, a):
		return abs(ord(self)-ord(a))

def solve(a):
	b = [string(s) for s in a]
	ans = 0
	for i in xrange(len(b)/2):
		ans += (b[i]-b[len(b)-i-1])
	return ans

for _ in xrange(input()):
	print solve(raw_input())